docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
```

Comment:
The existing installation sections are not all github based and are specifically for installing the operator and driver.
as far as I understand (@matancarmeli7  please confirm) this can be installed optionally after any installation type.
In any case, as this is a separate installation I believe that it should be documented separately but under the Installation chapter.

Reply:
@rivkap yes we can download it in any installation that we want

Reply:
the from a documentation point of view we keep it separate.

Reply:
why?
and either way, we should document per each type of installation

Reply:
is there a different way to install the host definer?

Reply:
I'll let @matancarmeli7 fill you in with the exact details (I haven't done myself)

Reply:
updated per meeting with @roysahar